cartoframes/data/dataset/dataset.py

```diff
@@ -230,7 +218,7 @@ def download(self, limit=None, decode_geom=False, retry_times=DOWNLOAD_RETRY_TIM
 
         return self._strategy.download(limit, decode_geom, retry_times)
 
-    def upload(self, with_lnglat=None, if_exists=FAIL, table_name=None, schema=None, credentials=None):
+    def upload(self, with_lnglat=None, if_exists=BaseDataset.FAIL, table_name=None, schema=None, credentials=None):
```

Comment:
I don't see why move constants to `BaseDataset`.  `BaseDataset` is an internal class, a user should never use it. I would prefer to use the constants from `Dataset`.  Maybe I am losing something

Reply:
👍 In this case, let's move it back to Dataset but then they should be documented